<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWin</class>
 <widget class="QDialog" name="SettingsWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ayarlar</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <layout class="QFormLayout" name="formLayoutSettings">
     <item row="0" column="0">
      <widget class="QLabel" name="labelLanguage">
       <property name="text">
        <string>Dil:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="langBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHotkey">
       <property name="text">
        <string>Varsayılan Hotkey:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="layoutHotkey">
       <item>
        <widget class="QLineEdit" name="lineHotkey"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelectHotkey">
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/assets/icons/select.svg</normaloff>:/assets/icons/select.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelTheme">
       <property name="text">
        <string>Tema:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="themeBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPatchNotes">
       <property name="text">
        <string>Patch Notes:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="plainPatchNotes">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Kaydet</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/assets/icons/save.svg</normaloff>:/assets/icons/save.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDiscard">
       <property name="text">
        <string>Vazgeç</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/assets/icons/cancel.svg</normaloff>:/assets/icons/cancel.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerSettings">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelVersion">
       <property name="text">
        <string>v2.0-beta1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
